packages/connectivity/src/scp-cf/destination/get-subscriber-token.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('getSubscriberToken()', () => {
     expect(serviceTokenSpy).toHaveBeenCalledWith(
       'destination',
       expect.objectContaining({
-        jwt: { iss: onlyIssuerXsuaaUrl }
+        jwt: { ext_attr: { zdn: 'subscriber-only-iss' } }
```

Comment:
Previously we mimicked a token with an issuer here. I changed this to be reflected in the `zdn` property, because the issuer should only be checked in the case we have an XSUAA token. If it is an IAS token, the subdomain of the tenant cannot be parsed from the issuer. Therefore we now first look at the `zdn` property and only if this is not there look at the issuer, if this is an XSUAA token.